doc/guides/building-node-with-ninja.md

```diff
@@ -1,28 +1,36 @@
 # Building Node with Ninja
 
-The purpose of this guide is to show how to build Node.js using [Ninja][], as doing so can be significantly quicker than using `make`. Please see [Ninja's site][Ninja] for installation instructions (unix only).
+The purpose of this guide is to show how to build Node.js using [Ninja][], as
+doing so can be significantly quicker than using `make`. Please see
+[Ninja's site][Ninja] for installation instructions (unix only).
 
 To build Node with ninja, there are 3 steps that must be taken:
 
 1. Configure the project's OS-based build rules via `./configure --ninja`.
-2. Run `ninja -C out/Release` to produce a compiled release binary.
-3. Lastly, make symlink to `./node` using `ln -fs out/Release/node node`.
+1. Run `ninja -C out/Release` to produce a compiled release binary.
+1. Lastly, make symlink to `./node` using `ln -fs out/Release/node node`.
+
+When running `ninja -C out/Release` you will see output similar to the following
```

Comment:
While we're here, can you add a comma after `ninja -C out/Release`

Reply:
I would rather keep this PR only about stylistic changes.

Reply:
My point is that this change has renumbered the steps so that instead of steps 1-10 it now reads steps 1-3, 1, 1-6.